### DISCUSSION TITLE
Relation manager translatable

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#12278] [Help]
### Package

Panel builder

### Package Version

v3.2.63

### How can we help you?

Can relationship managers be translatable?

**Comment by leandrocfe** [comment]
Yes, I think.. what is the issue?

**Reply by ArtMin96** [reply]
I can't find information anywhere about how the relation manager can be translatable.

**Reply by ArtMin96** [reply]
There is a `Filament\Resources\RelationManagers\Concerns\Translatable` trait for relationship managers, but I don't know how to do the rest.

**Reply by ArtMin96** [reply]
Sorry, I found this in the [Filament Spatie Translatable Plugin](https://github.com/filamentphp/spatie-laravel-translatable-plugin) plugin. Thanks
